plugins/modules/zos_blockinfile.py

```diff
@@ -96,13 +96,15 @@
   marker_begin:
     description:
     - This will be inserted at C({mark}) in the opening ansible block marker.
+    - Value needs to be different from marker_end.
     required: false
     type: str
     default: BEGIN
   marker_end:
     required: false
     description:
     - This will be inserted at C({mark}) in the closing ansible block marker.
+    - Required unique value different from marker_begin.
```

Comment:
seems kind of strange to have a different description here than in marker_begin. I'd say pick one or the other and use the same line in both module options

If you go with "Value needs to be different from marker_end." I'd say use stronger language "Value must be different from marker_end."